method/index.js

```diff
@@ -1,6 +1,6 @@
 const Init = require('./group/init')
 const AddMember = require('./group/add-member')
-const CreateLink = require('./link/create')
+const { CreateFeedGroupLink, CreateGroupSubgroupLink } = require('./link/create')
```

Comment:
good distinction. I think we should go further

if we split these into different specs : 
spec/link/feed-group
spec/link/group-subgroup (or something)

then we can delete these files and just have  linkFeedGroup.create and linkGroupSubroup.create

Reply:
Refactored it and it looks so much nicer! Got rid of a lot of files :D